pandas/core/frame.py

```diff
@@ -2988,6 +2988,64 @@ def to_orc(
             self, path, engine=engine, index=index, engine_kwargs=engine_kwargs
         )
 
+    @overload
+    def to_html(
+        self,
+        buf: FilePath | WriteBuffer[str],
+        columns: Sequence[str] | None = ...,
```

Comment:
Not related to your changes, but I think str is incorrect here, ints work too for example.

Reply:
Changed it